main/provider/helpers.ts

```diff
@@ -154,8 +154,8 @@ export function resError (errorData: string | EVMError, request: RPCId, res: RPC
 }
   
 export function getSignedAddress (signed: string, message: string, cb: Callback<String>) {
-  const signature = Buffer.from(signed.replace('0x', ''), 'hex')
-  if (signature.length !== 65) cb(new Error('Frame verifySignature: Signature has incorrect length'))
+  const signature = Buffer.from((signed || '').replace('0x', ''), 'hex')
```

Comment:
I'd put the default case in the signature above

Reply:
I wanted to but it doens't work because it can be `null` which wouldn't get set to an empty string

Reply:
ah yes, another reminder of why I try to avoid using `null`